uwblueprint/abtc
# Seb/volunteer signed up model - all inclusive

## Notion ticket link

[Create a volunteers signed up model](https://www.notion.so/uwblueprintexecs/95b4de36fae84be28f8c5de03758033f?v=5aeb0429d398432d8de8f5fa5a9823c5&p=8b868d9ad5d9435e9e22f9c8eb47c55c&pm=s)



## Implementation description
* Created data model for a user who is signed up



## Steps to test
1. Make a postman request to the api endpoint: '/users/volunteers-signed-up and check mongodb. There are special changes needed to make this work. It is detailed in the notion ticket.



## What should reviewers focus on?
* Any feedback?


## Checklist
